src/ontology/mondo-edit.obo

```diff
@@ -422271,17 +422271,18 @@ id: MONDO:0042974
 name: parainfluenza virus type 3 infectious disease
 def: "Parainfluenza virus type 3 is one of a group of common viruses known as human parainfluenza viruses (HPIV) that cause a variety of respiratory illnesses. Symptoms usually develop between 2 and 7 days from the time of exposure and typically resolve in 7-10 days. Symptoms may include fever, runny nose, and cough. HPIV-3 can also cause bronchiolitis, bronchitis, and pneumonia. Infants and young children are particularly susceptible to HPIV-3 infections, though older adults and those with a weakened immune system are also at risk for complications. HPIVs are usually spread from an infected person to others through coughing, sneezing, and/or touching. There is currently no vaccine to protect against parainfluenza virus infections. Most HPIV infections resolve on their own and do not require special treatment, though medical intervention may be necessary for severe breathing problems. Most adults have antibodies against parainfluenza but can get repeat infections." [GARD:0004215]
 subset: gard_rare
-synonym: "Human parainfluenza virus type 3" RELATED [GARD:0004215]
-synonym: "Human respirovirus 3 caused disease or disorder" EXACT [MONDO:patterns/specific_infectious_disease_by_agent]
-synonym: "Human respirovirus 3 disease or disorder" EXACT []
-synonym: "Human respirovirus 3 infectious disease" EXACT []
-synonym: "infection caused by human parainfluenza virus 3" RELATED []
-synonym: "infection caused by parainfluenza virus 3" RELATED []
-synonym: "infection due to human parainfluenza virus 3" EXACT []
-synonym: "infection due to parainfluenza virus 3" RELATED []
+synonym: "human parainfluenza virus type 3" RELATED [GARD:0004215]
+synonym: "human respirovirus 3 caused disease or disorder" EXACT [MONDO:patterns/specific_infectious_disease_by_agent]
+synonym: "human respirovirus 3 disease or disorder" EXACT []
+synonym: "human respirovirus 3 infectious disease" EXACT []
```

Comment:
Synonyms on lines 422275-422277 were not added by me, I just made them lower case/

Reply:
thanks!